### DISCUSSION TITLE
Error when deploying

### DISCUSSION THREAD

**Discussion by arnaudvdvk** [jkranz-rk/RecordTypePicker#32] [Q&A]
Hello,

Thanks for this very nice feature ! Unfortunalety, I'm facing error when deploying LWC "recordTypePicker".

I always get this error : Exception while getting the LWC bundle for reference rtp-record-type-picker-cpe of type component in file recordTypePicker.js-meta.xml

Anyone to help ? 👼 
Thanks !

**Comment by jkranz-rk** [comment]
Hi @arnaudvdvk 👋

Are you deploying the project metadata as-is from the repo, or installing the unlocked package?

**Reply by arnaudvdvk** [reply]
Hello @jkranz-rk , sorry, I just got to see your answer.
I'm deploying as-is from the repo.

Thanks for helping me,
Arnaud
